### DISCUSSION TITLE
Following examples, only errors

### DISCUSSION THREAD

**Discussion by GronbergInfo** [ceccopierangiolieugenio/pyTermTk#276] [Q&A] [answered]
Been testing out som examples, but when i run it, i only get errors that i cant find a solution to.
```
Traceback (most recent call last):
  File "D:\Python\a\a.py", line 7, in <module>
    root = ttk.TTk()
           ^^^^^^^^^
  File "D:\Python\a\.venv\Lib\site-packages\TermTk\TTkCore\ttk.py", line 116, in __init__
    w,h = TTkTerm.getTerminalSize()
    ^^^
TypeError: cannot unpack non-iterable NoneType object
ERROR: [WinError 6] The handle is invalid

Process finished with exit code 1

```

pip3 install --upgrade pyTermTk
Requirement already satisfied: pyTermTk in d:\python\a\.venv\lib\site-packages (0.40.0a1)

Any suggestions? All indicates that python and TTkTerm is installed correctly to me.

The code i copied from [here](https://github.com/ceccopierangiolieugenio/pyTermTk/blob/main/tutorial/helloworld/helloworld.001.py) that i even fail to run..

```
#!/usr/bin/env python3
#
import TermTk as ttk

root = ttk.TTk()
ttk.TTkLabel(parent=root, pos=(5,2), text="Hello World")
root.mainloop()
```

**Comment by ceccopierangiolieugenio** [comment]
that's odd,
are you running this example on windows?
I have to admit that I am not testing regularly it on windows because I need to run it on a VM.
Let me try.
in the meantime you can try it using WSL.

**Reply by GronbergInfo** [reply]
> that's odd, are you running this example on windows? I have to admit that I am not testing regularly it on windows because I need to run it on a VM. Let me try. in the meantime you can try it using WSL.

yes, on windows 10 ltsc, aiming to make a TUI for debian/linux.

**Reply by ceccopierangiolieugenio** [reply]
I think you are not running it inside a terminal,
I just ran it on inside both hyperTerminal and windows cmd and it works fine.
I think I will publish some instructions on how to run it under windows

**Reply by ceccopierangiolieugenio** [reply]
https://github.com/user-attachments/assets/0b48e325-8f26-492e-bcff-3ffc26084d6a

**Reply by ceccopierangiolieugenio** [reply]
the only issue I am having is that CTRL-C is not recognised as sigint so the app does not exit (unless the quit method is called inside the app)

**Reply by ceccopierangiolieugenio** [reply]
ok, on windows you can use CTRL-BREAK or CTRL-PAUSE to interrupt it

**Comment by ceccopierangiolieugenio** [comment]
https://github.com/user-attachments/assets/40ade33c-93db-41e8-ba68-e8917b1f83c5

**Comment by ceccopierangiolieugenio** [comment, marked as answer]
[Windows10 Install Python3 and pyTermTk using PowerShell in less than 2 min.](https://www.youtube.com/watch?v=1nQ3P3bgKNY)

[![Windows10 Install Python3 and pyTermTk using PowerShell in less than 2 min.
](https://img.youtube.com/vi/1nQ3P3bgKNY/0.jpg)](https://www.youtube.com/watch?v=1nQ3P3bgKNY)

https://www.youtube.com/watch?v=1nQ3P3bgKNY
